src/chat/menu/controller.js

```diff
@@ -24,4 +50,29 @@ module.exports = class MenuController {
             }
         }
     }
+
+    myProfile() {
+        this.cardService.profileScreen = true;
+        this.cardService.archiveScreen = false;
+        this.cardService.uptodateScreen = false;
+        this.cardService.searchScreen = false;
+        this.cardService.recycleBinScreen = false;
+        this.cardService.userScreen = false;
+        this.uptodateScreen = false;
+        this.cardService.trigger('UptodateScreen');
+        this.cardService.trigger('Profile');
+    }
+
+    shoppingList() {
+        this.uptodateScreen = true;
+        this.profileScreen = false;
+        this.cardService.profileScreen = false;
+        this.cardService.archiveScreen = false;
+        this.cardService.uptodateScreen = true;

```

Comment:
I wrote about this before on Gizem's PR. Please fix this logic. Insteas of keeping separate variables for each screen, keep only one variable that holds the screen name.